src/App.tsx

```diff
@@ -2,18 +2,24 @@ import { Navigate, Route, Routes } from "react-router-dom";
 import PlantDetailPage from "./pages/plantDetailPage/PlantDetailPage.tsx";
 import MyCollectionPage from "./pages/myCollectionPage/MyCollectionPage.tsx";
 import AddNewPlant from "./pages/addNewPlantPage/AddNewPlant.tsx";
+import GlobalStyle from "./theme/globalStyles.ts";
+import { theme } from "./theme/theme.ts";
+import { ThemeProvider } from "styled-components";
 
 function App() {
   return (
     <>
-      <Routes>
-        <Route path={"/"} element={<MyCollectionPage />} />
-        <Route path={"/dodaj-roślinkę"} element={<AddNewPlant />} />
-        {/*if I'm rendering Routes below single plant component
+      <ThemeProvider theme={theme}>
+        <GlobalStyle />
+        <Routes>
+          <Route path={"/"} element={<MyCollectionPage />} />
+          <Route path={"/dodaj-roślinkę"} element={<AddNewPlant />} />
+          {/*if I'm rendering Routes below single plant component
```

Comment:
we will look into the routing separately at some point - you can remove the comment for now :)

Reply:
removed! I was keeping it to not forget why I had a star there.